### DISCUSSION TITLE
Need to remove the black square pixel

### DISCUSSION THREAD

**Discussion by anieshvishnu** [ImageMagick/ImageMagick#3839] [Help]
I need the remove the black square pixel and get the text-only.
![Capture](https://user-images.githubusercontent.com/24503245/122417118-e34e1c80-cfa6-11eb-8555-c664fe70fce0.PNG)

The above image is zoomed the pixel are really small.

tried this
  ` convert 1.png -morphology thicken '1x3>:1,0,1' result.png
`

**Comment by snibgo** [comment]
I'm not sure what you want. Perhaps you want to turn isolated black pixels, that make "ff", white.

Please show the actual image file, not zoomed. Also: what version of IM, on what platform?

**Reply by anieshvishnu** [reply]
![Capture1](https://user-images.githubusercontent.com/24503245/122423694-be0fdd00-cfab-11eb-8819-b11036a4a466.PNG)
I want to remove this

**Reply by anieshvishnu** [reply]
original image
![orig](https://user-images.githubusercontent.com/24503245/122424134-1941cf80-cfac-11eb-8cca-11a8ae90ff93.PNG)

**Comment by snibgo** [comment]
Your "original image" has gray pixels making "ffi". So I don't understand what you want.

**Reply by anieshvishnu** [reply]
I want to remove the ffi

**Comment by snibgo** [comment]
Those pixels are gray, not black.

**Reply by anieshvishnu** [reply]
when you zoom it. It is black

**Comment by snibgo** [comment]
If you download the image you uploaded, and zoom in, you will see the pixels are gray.
![x](https://user-images.githubusercontent.com/33812368/122472064-f9180d80-cfb7-11eb-93f0-1a32fdf57018.png)

Perhaps you are showing screenshots of an image file, with different screenshots covering different areas at different scales.

Please show the original image file.
